src/mds/MDCache.cc

```diff
@@ -12035,6 +12035,20 @@ void MDCache::show_subtrees(int dbl)
     }
   }
 
+  if (subtrees_seen.size() > SUBTREES_COUNT_THRESHOLD && depth >
+      SUBTREES_DEPTH_THRESHOLD) {
+    dout(15) << "number of subtrees > SUBTREES_COUNT_THRESHOLD and depth > "
+		"SUBTREES_DEPTH_THRESHOLD; not printing subtrees" << dendl;
```

Comment:
Let's at least print something a little useful like  the number of subtrees in this line. I think we can forgo printing "subtrees > SUB..." stuff.